lib/Ix/DBIC/ResultSet.pm

```diff
@@ -799,6 +875,21 @@ sub ix_update ($self, $ctx, $to_update) {
   return \%result;
 }
 
+=method ix_destroy($ctx, $to_destroy)
+
+This method implements the destroy logic for 'Foo/set'; you should not ever
+need to call it independently. It is passed an arrayref of ids to destroy,
+does error checking, calls hooks, then runs the database update. It calls any
+postprocessing hooks, and returns a hashref that is eventually used to
+generate the L<Ix::Result> in response to the Foo/set call.
+
+I<IMPORTANT>: C<ix_destroy> does I<not> delete rows: destroying a record sets
+its C<isActive> property to null, its C<dateDestroyed> property to the current
+timestamp, and updates its C<modSeqChanged>. To delete rows entirely, use
+C<ix_purge>.
```

Comment:
Maybe also mention what it means for an object to be destroyred. That is, it appears for all intents and purposes to no longer exist, but since the row remains around we can generate ix_changes information about it

Reply:
Noted, thanks.